src/main/java/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineView.java

```diff
@@ -606,11 +606,15 @@ private int triggerBuild(final AbstractProject<?, ?> triggerProject, final Abstr
         if (manualTrigger != null) {
             final Set<String> downstreamProjectsNames =
                     Sets.newHashSet(Splitter.on(",").trimResults().split(manualTrigger.getDownstreamProjectNames()));
-            if (downstreamProjectsNames.contains(project.getFullName())) {
-                configs = manualTrigger.getConfigs();
+            for (String downstreamProjectsName : downstreamProjectsNames) {
+                final Item item = Jenkins.getInstance().getItem(downstreamProjectsName, project, Item.class);
+                if (item == project) {
+                    configs = manualTrigger.getConfigs();
+                    break;
+                }
             }
         }
-
+        
         final BuildTrigger autoTrigger = upstreamProjectPublishersList.get(BuildTrigger.class);

```

Comment:
As i see in debugger build is not triggered because BuildTrigger.class is imported from wrong package